src/ucp/wireup/wireup.c

```diff
@@ -636,7 +636,7 @@ ucp_wireup_process_request(ucp_worker_h worker, ucp_ep_h ep,
 static unsigned ucp_wireup_send_msg_ack(void *arg)
 {
     ucp_ep_h ep = (ucp_ep_h)arg;
-    ucp_rsc_index_t rsc_tli[UCP_MAX_LANES];
+    ucp_lane_index_t rsc_tli[UCP_MAX_LANES];
```

Comment:
this array is resource index, not lane index
so entry value is ucp_rsc_index_t

Reply:
seems it is used to map local to remote lanes:
https://github.com/openucx/ucx/blob/a0d67ac65da4b42ed288e992ff8dccdbdbe8b09a/src/ucp/wireup/wireup.c#L344

Reply:
are you sure? this is rsc_tli, not lanes2remote

Reply:
In line 646 it is passed as 4th argument to `ucp_wireup_msg_send`.
And in line 220 we can see that the meaning of the 4th argument:
`ucp_wireup_msg_send(ucp_ep_h ep, uint8_t type, const ucp_tl_bitmap_t *tl_bitmap,
                    const ucp_lane_index_t *lanes2remote)`

Reply:
i think we can pass NULL instead of rsc_tli and remove it altogether, since tl_bitmap is 0 the worker address is empty anyway for UCP_WIREUP_MSG_ACK message.
@evgeny-leksikov do you agree?

Reply:
> @evgeny-leksikov do you agree?

yes